### PR TITLE
Created Person-IDs for donors of BMLacq811 (DamaKrestos and Mikael)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
The note in BMLacq811 (according to Marrassini) says: ዝመጽሐፍ፡ ዘዘ<persName ref="PRS13981Mikael">ሚካኤል፡</persName>
ወዘአቡሁ፡ <persName ref="PRS13980DamaKrestos">ደመ፡ ክርስቶስ፡</persName> ዘሰረቆ፡ ወዘፈሐቆ፡ ውጉዘ፡ ለይኩን፡ I understand, that Dama Krestos is the father of Mikael and together they donated this book to receive pardon for whatever they did. I selected "DonationNote" in the description of BMLacq811, but in the person document I could not find donation or donor and selected "Owner" for that reason. Is that correct?

### CODE REVIEW

File: new/PRS13980DamaKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደመ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dama Krəstos</persName>
+                    <floruit>19th or 20th century</floruit>
+                    <listRelation>
+                        <relation name="saws:isParentOf" active="#PRS13980DamaKrestos" passive="#PRS13981Mikael"/>
```
Comment (eu-genia) [inline new/PRS13980DamaKrestos.xml:52]: I know it is confusing, but `saws `relations are used for works/texts and manuscripts, for person to person relations we use `snap `and `betmas`-prefixed relations. In this case probably `name="snap:FamilyOf"` or `snap:KinOf`

Comment (eu-genia) [reply new/PRS13980DamaKrestos.xml:52]: see SAWS ontology definitions here https://ancientwisdoms.ac.uk/method/ontology/
```
<AnnotationAssertion>
<AnnotationProperty abbreviatedIRI="rdfs:comment"/>
<IRI>#isParentOf</IRI>
<Literal xml:lang="en" datatypeIRI="http://www.w3.org/1999/02/22-rdf-syntax-ns#PlainLiteral">The parent 
OriginalMaterial is a direct source for the production of part or 
all of the child OriginalMaterial</Literal>
</AnnotationAssertion>
```

File: new/PRS13981Mikael.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሚካኤል</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Mikāʾel</persName>
+                    <floruit>19th or 20th centuries</floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:isDescendantOf" active="#PRS13981Mikael" passive="PRS13980DamaKrestos"/>
```
Comment (eu-genia) [inline new/PRS13981Mikael.xml:53]: again, saws ontology is for texts/manuscripts, not persons
```
<AnnotationAssertion>
<AnnotationProperty abbreviatedIRI="rdfs:comment"/>
<IRI>#isDescendantOf</IRI>
<Literal xml:lang="en" datatypeIRI="http://www.w3.org/1999/02/22-rdf-syntax-ns#PlainLiteral">An ancestor 
OriginalMaterial is a source material (direct or indirect) for the production of part or all of the descendant 
OriginalMaterial</Literal>
</AnnotationAssertion>
```
use a snap relation please

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13980DamaKrestos.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS13980DamaKrestos.xml:52]: if he is father, then use FatherOf relation `snap:FatherOf`

Comment (eu-genia) [reply new/PRS13980DamaKrestos.xml:52]: In the MS record, you should use donor type
`persName role="donor" ref="PRS0000"`

File: new/PRS13980DamaKrestos.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                        <relation name="saws:hasOwned" active="#PRS13980DamaKrestos" passive="BMLacq811"/>
```
Comment (eu-genia) [inline new/PRS13980DamaKrestos.xml:53]: since he has not owned the manuscript use `lawd:hasAttestation `relation

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
I think that the name is Zazamikāʾel (referring to Zamikāʾel ʾAragāwi)?

**Comment by thea-m** [comment]
Zamikāʾel I mean, sorry

File: new/PRS13981Mikael.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13981Mikael.xml:53]: Done.

Comment (CarstenHoffmannMarburg) [reply new/PRS13981Mikael.xml:53]: snap:SonOf

File: new/PRS13980DamaKrestos.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13980DamaKrestos.xml:52]: changed to snap:FatherOf

File: new/PRS13980DamaKrestos.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13980DamaKrestos.xml:53]: Done.

**Comment by CarstenHoffmannMarburg** [comment]
If we agree on Zämikaʾel instead of Mikaʾel, I need to create a new PRS-ID. Correct?

File: new/PRS13980DamaKrestos.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13980DamaKrestos.xml]: Changed donation note in BMLacq811.

**Comment by eu-genia** [comment]
Yes, the name is Zamikael, and no, you do not need a new Person ID, you only change the name in the record 13981 itself

**Review by eu-genia — CHANGES_REQUESTED** [review]
please DO NOT change the person, NOT xml:id, NOT filename. the file name and xml:ID MUST remain as they were, also in the relation.

**Comment by CarstenHoffmannMarburg** [comment]
I restored ID PRS13981Mikael for the person Zamikāʾel, son of Dama Krəstos, mentioned in BMLacq811. I wonder why should not create a new ID, since now the PRS-ID does not correspond to the name any more.

**Comment by eu-genia** [comment]
The ID is just a string, it is not supposed to have any meaning at all. So you were supposed only to correct the spelling of the  name inside, but not change anything structural.

**Comment by CarstenHoffmannMarburg** [comment]
OK.

File: new/PRS13981Mikael.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘሚካኤል</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamikāʾel</persName>
+                    <floruit>19th or 20th centuries</floruit>
+                </person>
+                <listRelation>
+                    <relation name="snap:SonOf" active="#PRS13981Zamikael" passive="PRS13980DamaKrestos"/>
```
Comment (eu-genia) [inline new/PRS13981Mikael.xml:54]: the link here is still wrong it should correspond to the xml:id of the entry, and have no # in front of it

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13981Mikael.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13981Mikael.xml]: I corrected the xml:ID in PRS13981Mikael yesterday but probably did not push it orderly. I did it once again for PRS13981Mikael and for PRS13980DamaKrestos and I hope I did right.

File: new/PRS13981Mikael.xml [hunk 5]
```diff
@@ -5,7 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title>Zamikāʾel</title>
+                <title>Mikāʾel</title>
```
Comment (eu-genia) [inline new/PRS13981Mikael.xml]: the title should correspond to the name ) it is not the ID

**Review by eu-genia — APPROVED** [review]

File: new/PRS13982Estifanos.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-02-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ʾƎsṭifanos</persName>
```
Comment (eu-genia) [inline new/PRS13982Estifanos.xml:48]: sorry here the fidel and the transcription must be switched

File: new/PRS13982Estifanos.xml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">እስጢፋኖስ</persName>
+                    <floruit>19th or 20th centuries</floruit>
```
Comment (eu-genia) [inline new/PRS13982Estifanos.xml:50]: mark up floruit (notBefore notAfter)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13981Mikael.xml [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+            <change when="2023-02-15" who="CH">Updated title</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘሚካኤል</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamikāʾel</persName>
+                    <floruit>19th or 20th centuries</floruit>
```
Comment (eu-genia) [inline new/PRS13981Mikael.xml:51]: mark up here too `<floruit notBefore="1800" notAfter="1950">` etc

File: new/PRS13980DamaKrestos.xml [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+            <change who="CH" when="2023-02-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደመ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dama Krəstos</persName>
+                    <floruit>19th or 20th century</floruit>
```
Comment (eu-genia) [inline new/PRS13980DamaKrestos.xml:50]: mark up floruit

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13981Mikael.xml (hunk 8 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13981Mikael.xml:51]: Done. I marked up in the requested way and kept the phrase "19th or 20th centuries"

File: new/PRS13982Estifanos.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13982Estifanos.xml:50]: Done. I marked up in the requested way and kept the phrase "19th or 20th centuries"

File: new/PRS13980DamaKrestos.xml (hunk 9 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13980DamaKrestos.xml:50]: Done. I marked up in the requested way and kept the phrase "19th or 20th centuries"

File: new/PRS13981Mikael.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13981Mikael.xml]: Understand. Done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**